README.md

```diff
@@ -5,19 +5,73 @@
+
+* A ```post.rb``` model file that will contain: validations, database relationships, callbacks, and any custom logic for posts.
+
+* A ```posts_controller.rb``` file that will have methods that will manage data flow for the Post behavior, this will include the full set of CRUD features, the standard methods are: index, new, create, show, edit, update, and destroy.
+
+* A ```views/``` directory that will contain a corresponding view for each of the pages that the end user will access. For a CRUD based model, a few of the standard views would include: an index view to show all records, a show page that shows a specific record, and then new and edit pages that both render a form.
+
+## Request Flow
+![MVC Request Flow](http://reif.io/lib/flatiron/mvc-flow.jpg)
+
+## Roles and Responsibilities
+
+### Models
+
+At the end of the day the model file is a Ruby class. If it has a corresponding database table it will inherit from the ActiveRecord::Base class, which means that it has access to a number of methods that assist in working with the database. However, you can treat it like a regular Ruby class, allowing you to create methods, data attributes, and everything else that you would want to do in a class file. In a typical model file you will find: custom database query scopes, default settings for database columns, validations, model-to-model relationships, callbacks, and custom algorithms related to that model.
+
+It is important to remember to follow the single responsibility principle for your model class files. If any of the methods that you place in the model file perform tasks outside the scope of that specific model, they should probably be moved to their own class.
+
+### Controllers
+
+As mentioned before, the controller is like the waiter in a restaurant. The controllers connect the models, views, and routes. To make the process more straightforward, think in terms of the following process:
+
+* The view looks to the controller and only has access to the instance variables that the controller makes available. Those instance variables will contain any/all data coming in from the database.
+* The routes file looks to the controller and ensures that the methods in the controller match the items in the routes file.
+* The controller methods also handle the redirect logic for the application, for example, when a new record is created, the controller dictates where the user should be redirected after the new entry is successfully added to the database, along with what confirmation message is sent to the user.
+
+The controllers can also help to protect against security issues, in Rails 4+ the controller forces all attributes submitted through forms to be declared as a strong parameter.
```

Comment:
This is super specific, and I don't think it's important for students to be thinking about security just yet. I'd either drop this sentence or just remind them that the controller is like a waiter in that not all inputs from patrons make it back to the kitchen